src/stravalib/model.py

```diff
@@ -936,6 +941,33 @@ class Activity(
     _naive_local = field_validator("start_date_local")(naive_datetime)
 
 
+class ClubAthlete(strava_model.MetaAthlete):
```

Comment:
Ok @jsamoocha i went a little off plan here because i wasn't sure how to best do this.

the spec says that club activities should contain a metaAthlete object. But metaAthlete only has the id field but the spec says this should return first name and last initial. 

so to follow and allow the user to get the same thing that the actual API returns, i created this club athlete object. if this already exists or is documented somewhere i must be missing it. this allows for us to return the data exactly as strava returns it.

Reply:
Have you considered inheriting (or simply reusing) `strava_model.ClubAthlete`? It has the same fields. This looks like another inconsistency in the Strava API, where `ClubActivity.athlete` should be of type `ClubAthlete` (instead of `MetaAthlete`).

This, however, will trip up mypy because `ClubAthlete` is not a subtype of `MetaAthlete` (so it will require a `# type: ignore[assignment]`).